tests/getLikesMock.test.js

```diff
@@ -0,0 +1,19 @@
+import { jestAPICredentials } from '../__mocks__/jestAPICredentials.js';
+import { getLikesMock } from '../__mocks__/getLikesMock.js';
+
+// arrange
+describe('Tests fetch with axios', () => {
+  const { LIKES_ENDPOINT_API } = jestAPICredentials;
+  const endpoint = LIKES_ENDPOINT_API;
+
+  it('Grabs Likes Count', async () => {
+    const likesData = await getLikesMock(endpoint);
+    let likeCount = '';
+    likesData.forEach((likeDataObject) => {
```

Comment:
It's nice idea to run a forEach function to get the like count of specific item inside the API endpoint.